man/journald.conf.xml

```diff
@@ -402,6 +402,18 @@
         this option is enabled by default, it is disabled in all others.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>Audit=</varname></term>
+
+        <listitem><para>Takes a boolean value. If enabled <command>systemd-journal</command> will turn on
+        kernel auditing on start-up. If disabled it will turn it off. If unset it will neither enable nor
+        disable it, and leave it the way it is configured. Note that this option does not control whether
+        <command>systemd-journald</command> collects generated audit records, it just controls whether it
+        tells the kernel to generate them. This means if another tool turns on auditing even if
+        <command>systemd-journald</command> left it off, it will still collect the generated
+        messages. Defaults to on.</para></listitem>
```

Comment:
Hmm, that seems very complicated. If Audit=off here, I'd expect journald to not collect messages, period.
And if the described behaviour is to be retained, then the option needs to be renamed, to `EnableKernelAudit=` or so.

Reply:
> Hmm, that seems very complicated. If Audit=off here, I'd expect journald to not collect messages, period.

Well, that's not easy to implement, since the socket for it is allocated by systemd already, before journald is started.

I explicitly documented this behaviour to make this understandable.

> And if the described behaviour is to be retained, then the option needs to be renamed, to EnableKernelAudit= or so.

Well, it's an option that can be used to disable audit too. EnableKernelAudit= is a misnomer, since if you set it to off you'd assume we wouldn't enable it, but actually we will explicitly disable it, which is stricter.

Reply:
(not bound at all to the naming here, but i couldn't come up with anything better)

Reply:
My understanding is that you are both actually thinking of a choice-type, not a boolean/tristate.

Smashing your suggestions together, `ConfigureKernelAudit=<empty>|enable|disable|redirect` makes sense to me.